tests/test_preprocessing/test_columncapper.py

```diff
@@ -0,0 +1,108 @@
+import pytest
+import numpy as np
+import pandas as pd
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import StandardScaler
+from sklearn.utils import estimator_checks
+from sklearn.utils.estimator_checks import check_transformers_unfitted
+from sklearn.utils.validation import FLOAT_DTYPES
+
+from sklego.common import flatten
+from sklego.preprocessing import ColumnCapper
+from tests.conftest import nonmeta_checks
+
+# ColumnCapper works with nan/inf cells
+nonmeta_checks_allow_nan_inf = [test for test in nonmeta_checks if test.__name__ != 'check_estimators_nan_inf']
+
+
+@pytest.mark.parametrize("test_fn", flatten([
```

Comment:
Wouldn't failing at some test block the PR? Or are you saying that you would manually merge it even with failed tests?

Reply:
Ah yes, I should've been more clear. If possible, it's best to use the lists containing all tests for a specific category of transformer. 

If that isn't possible because on of the tests in that list can't pass, copy the contents of the list to the test parameterization and put a comment in the body to indicate which test you removed and why.